### PR TITLE
Frontend appliances

### LINKED ISSUE #6: Project Create
Het aanmaken van een project op de project pagina.
Gebruik makend van userIds (deze kunnen nu nog 'fake' zijn)
Zorgen dat de kleur niet in # zichtbaar is maar pak zoals in de table : circles => clickable => project color picked 👍🏻 (zorg dat gebruiker ziet welke keur gekozen is)
Voorbeeld : 
image

Name => input field
VALIDATION in frontend too.
image

Summary:
   Name: input field, validation
   Color: clickable color circles
   Users: Just use numbers for now. Find a way to be able to select more then 1 number.  Maybe a dropdown compont library?

### PR DESCRIPTION
**Pull request by YesmanYasir**
had to fix some merge conflicts hopefully alles gefixt nu validation is nog ni klaar lmk als het slecht is

### CODE REVIEW

**Comment by RoelCrabbe** [comment]
npm install react-select use componets library's to help u

**Comment by RoelCrabbe** [comment]
Used : `npm install --save @fortawesome/react-fontawesome @fortawesome/free-solid-svg-icons` and `npm install react-select`

For our final result of : 
image

**Review by YesmanYasir — COMMENTED** [review]
niceu

File: back-end/types/index.ts [hunk 1]
```diff
@@ -6,9 +6,7 @@ enum Color {
     Blue = '#0000FF',
     Yellow = '#FFFF00',
     Orange = '#FFA500',
-    Purple = '#800080',
-    Black = '#000000',
```
Comment (RoelCrabbe) [inline back-end/types/index.ts:10]: White & black are colors that are not allowed to be used here

**Review by RoelCrabbe — APPROVED** [review]

**Pull request merged**